worklets/alien.json

```
@@ -0,0 +1,28 @@
+{
+  "workletName": "AlienPainter",
+  "packageName": "houdini-alien",
+  "npmUrl": "https://www.npmjs.com/package/houdini-alien",
+  "cdnUrl": "https://unpkg.com/houdini-alien@0.1.1/AlienPainter.js",
+  "demoUrl": "https://andreban.github.io/houdini-alien/index.html",
+  "tags": ["paint"],
+  "customProps": {
+    "--alien-bg-color": {
+      "type": "color",
+      "default": "#000000"
+    },
+    "--alien-fg-color": {
+      "type": "color",
+      "default": "#FFFFFF"
+    },
+    "--alien-grid-size": {
+      "type": "length",
+      "default": "1px"
```

```suggestion
      "default": "2"
```

I think this works without the px. Also `2` has a little less visual vibration

---

Sounds good to me. Applied.